docs/sources/panels-visualizations/query-transform-data/_index.md

```diff
@@ -108,74 +116,121 @@ SELECT hostname FROM host WHERE region IN($region)
 query_result(max_over_time(<metric>[${__range_s}s]) != <state>)
 ```
 
+### Query library
+
+{{< admonition type="note" >}}
+This feature is only available on Grafana Enterprise and Grafana Cloud.
+{{< /admonition >}}
+
+{{< docs/private-preview product="Query library" >}}
+
+The query library lets you save queries that you've created so you can reuse them later:
+
+{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-query-library-v11.6.png" max-width="550px" alt="" >}}
+
+In the **Query library** drawer, you can:
+
+- Search for queries by data source name, query content, and description.
+- Filter by data source name and author name (filters use the OR operator).
+- Edit a query description.
```

Comment:
you can also delete queries from the query library here - think this is worth mentioning

Reply:
Good call!